### ISSUE TITLE
Request to extend the BTP Service Operator to support multiple BTP regions for a single K8s namespace

### ISSUE THREAD

**Issue by valiparsa** [SAP/sap-btp-service-operator#278]
#### What happened?

`SAP BTP Service Operator` has a tight dependency to the K8s `namespaces`.
As mentioned [in the multitenancy section](https://github.com/SAP/sap-btp-service-operator#multitenancy) there is a one-to-one dependency between a BTP `subaccounts` and a K8s `namespaces`. More precisely, **all** `serviceinstance` and `servicebinding` objects in one k8s `namespace` will lead to instances and binding keys in **one and only one** BTP `subaccount`.

This approach would be problematic for the components which are moving toward supporting multiple BTP regions to achieve cost optimization. Which means they expect service keys from different BTP regions to be stored in different K8s `secrets` in **one** K8s `namespace`.

#### Affected versions

All available versions up to and including [v0.4.1](https://github.com/SAP/sap-btp-service-operator/releases/tag/v0.4.1)

#### What would you expect to happen?

We expect `BTP Service Operator` to use different `Service Manager` service keys even in one K8s `namespace` for different `serviceinstance` and `servicebinding` objects in one K8s `namespace`. 
For instance information about the target `Service Manager` service key can be mentioned in the `serviceinstance` and `servicebinding` objects so that `BTP Service Operator` uses the mentioned `Service Manager` service key to issue the instance and the binding key.

**Comment by kerenlahav** [comment]
We have a feature request to include a secret name in service instance which we are currently evaluating.
I'll update here once have the timelines.

**Comment by valiparsa** [comment]
@kerenlahav thanks a lot for your response and thanks for considering this feature request.

**Comment by ptesny** [comment]
> We have a feature request to include a secret name in service instance which we are currently evaluating. I'll update here once have the timelines.

I was about to ask for it as well. It could be very useful to be able to access for instance same regional SAP HANA Cloud database instance or SAP Datasphere from any namespace in a single k8s cluster;

**Comment by valiparsa** [comment]
@kerenlahav the issue is going to be closed as it was stale for 30 days. Do you have any updates maybe?

**Comment by kerenlahav** [comment]
Hi,
it is planned for the beginning of Q4

**Comment by valiparsa** [comment]
@kerenlahav thanks for your reply. That would be very helpful if you can share the link to the created Backlog item via MS Teams chat or Slack with me so that I can share it with the team.

**Comment by kerenlahav** [comment]
I couldn't find you on teams, please reach out to Avi Lupu

**Comment by TalShorSap** [comment]
We are currently working on this feature, we hope it will be released soon.
We will update when relevant. Thx

**Comment by valiparsa** [comment]
@TalShorSap I saw [this PR](https://github.com/SAP/sap-btp-service-operator/pull/341) was merged into the master. Is this PR the implementation of the feature?

**Comment by kerenlahav** [comment]
hi @valiparsa 
although the pr is merged this feature is not yet released and won't be included in the next release, we are currently at war and cannot commit to timelines.
I will update once this feature is ready.
thanks
